pywwt/layers.py

```diff
@@ -70,26 +73,34 @@ def _add2dict(self, diction, full_layer, bandpass):
         while short in diction[bandpass]:
             if suffix:
                 suffix += 1
-                short += str(suffix)
             else:
                 suffix = 1
-                short += str(suffix)
+                
+            short += str(suffix)
 
         diction[bandpass][short] = {}
         diction[bandpass][short]['full_name'] = full_layer
         diction[bandpass][short]['thumbnail'] = None
```

Comment:
I think 'thumbnail' can be done away with unless there is a reason to keep all passable options to the JS API.  I can imagine this may be useful later on if:
- someone was making a pywwt GUI and wanted to use a thumbnail image as is used on the wwt web client. 
- someone wanted to send a custom layer from pywwt to the WWT webclient using this class

However, both are beyond the scope of this feature.

Reply:
OK, thanks for letting me know. We can keep it mind in case we widen this feature's scope in the future.